jdaviz/configs/specviz/plugins/line_analysis/tests/test_line_analysis.py

```diff
@@ -41,6 +44,50 @@ def test_plugin(specviz_helper, spectrum1d):
             assert len(result_dict.get('uncertainty')) > 0
 
 
+@pytest.mark.filterwarnings('ignore:No observer defined on WCS')
+def test_spatial_subset(cubeviz_helper):
+    """
+    Tests that the spatial selection returns any valid result, DOES NOT VALIDATE THE VALUE
+    Not checking the value attempts to circumvent the issue we ran into here:
+    https://github.com/spacetelescope/jdaviz/pull/1564#discussion_r949427663
+    """
+
+    flux = np.arange(250).reshape((5, 5, 10)) * u.Jy
+    wcs_dict = {"CTYPE1": "RA---TAN", "CTYPE2": "DEC--TAN", "CTYPE3": "WAVE-LOG",
+                "CRVAL1": 205, "CRVAL2": 27, "CRVAL3": 4.622e-7,
+                "CDELT1": -0.0001, "CDELT2": 0.0001, "CDELT3": 8e-11,
+                "CRPIX1": 0, "CRPIX2": 0, "CRPIX3": 0}
+    w = WCS(wcs_dict)
+    large_cube = Spectrum1D(flux=flux, wcs=w)
```

Comment:
Why is this called "large_cube"?

Reply:
The reason is that it's a slightly (larger) modified version of the default spectrum1d cube on our tests. I wanted to make it easier to make a spectral selection for the sake of this pr

Reply:
So it is more like "a little larger cube" instead of like "big data cube"?